crates/chain/src/rusqlite_impl.rs

```diff
@@ -533,10 +535,28 @@ impl keychain_txout::ChangeSet {
         )
     }
 
+    /// Get v1 of sqlite [keychain_txout::ChangeSet] schema
+    pub fn schema_v1() -> String {
+        format!(
+            "CREATE TABLE {} ( \
+            descriptor_id TEXT NOT NULL REFERENCES {}, \
+            spk_index INTEGER NOT NULL, \
+            spk BLOB NOT NULL, \
+            PRIMARY KEY (descriptor_id, index) \
```

Comment:
Small typo in column name.

```suggestion
            PRIMARY KEY (descriptor_id, spk_index) \
```